### PR TITLE
UHM-6576: rename and parameterize publish script to support publishin…

### PR DESCRIPTION
**Pull request by mogoodrich**
…g to stable and unstable

### CODE REVIEW

**Comment by mogoodrich** [comment]
@mseaton  my thought is that with this change, the "release" stage should be as simple as the following, unless I"m forgetting something:

${project.build.directory}/debian/publish.sh stable

**Comment by mogoodrich** [comment]
Oops... just realized I'll need to update this script to include "stable" or "unstable" in the filename, will work on that...

File: debian/publish.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # DEBIAN_REPO_PASSWORD='password'
 
 DEBIAN_REPO_BASE_URL="https://openmrs.jfrog.io/artifactory/deb-pih/pool"
-DEBIAN_REPO_DISTRIBUTION=unstable
+DEBIAN_REPO_DISTRIBUTION=$1
```
Comment (mseaton) [inline debian/publish.sh:8]: Might be nice to have a check to validate that either "unstable" or "stable" is passed in, and have usage documented around this.

Comment (mogoodrich) [reply debian/publish.sh:19]: Good call, updated

**Pull request merged**